scripts/referrer-user-count.ts

```diff
@@ -0,0 +1,74 @@
+import yargs from 'yargs'
+import { protocolFilters, supportedNetworkIds } from './consts'
+import { fetchReferralEvents, removeDuplicates } from './referrals'
+import { NetworkId, Protocol, protocols } from './types'
+import { writeFileSync } from 'fs'
+
+async function getArgs() {
+  const argv = await yargs
+    .env('')
+    .option('protocol', {
+      description: 'protocol that the referrals are for',
+      demandOption: true,
+      choices: protocols,
+    })
+    // This is coerced as a string array to prevent 0x... from being interpreted as a number
+    .option('referrerAddresses', {
+      description: 'referrer address(es)',
+      type: 'string',
+      demandOption: true,
+      coerce: (arg) => new Set(arg.split(',')),
+    })
+    .option('networkIds', {
+      description: 'Comma-separated list of network IDs',
+      type: 'string',
+      demandOption: false,
+      coerce: (arg: string) => {
+        const networkIds = arg.split(',').map((id) => id.trim())
+        const invalidIds = networkIds.filter(
```

Comment:
This is going to make it wierd to use the local testnet.  Is this check really necessary?

Reply:
@silasbw This user error could occur when someone types `celo-mainet` instead of `celo-mainnet`. With this error, the script chugs along fine, and the output file now omits the user's reference to `celo-mainnet`. I see two solutions for this:
- Leave it as is and create a middleware to validate network IDs used in CLI in a later PR.
- Remove the `networkIds` parameter entirely and the ability to query per network.

Reply:
Opting for option #1 for now.